### ISSUE TITLE
Book DAC test for new features added since last audit for census

### ISSUE THREAD

**Issue by yatesn** [ONSdigital/design-system#1908]
Features needing an accessibility audit include:

Session timeout warning
Checkboxes/radios
Conditionally revealed checkboxes and radios (issues around aria-extended and complexity)
Select all button
Borderless
Errors (not putting first option in focus after selecting error summary link)
Sticky table of contents nav
Download button
Mutually exclusive (check if we should announce that selecting a checkbox will uncheck the mutually exclusive checkbox. See: Checkboxes)
Sortable tables (buttons styled as links?)
Date/duration (label/legend changes)
legendIsQuestionTitle
Send/reply to messages
DS site search
Accordions/collapsible (links with button behaviour, button context)
Link hover styling
Textarea character counter audio feedback
Character limit on input
Warning panels (panel hidden prefix text)
Cookies
Table of contents (skip over nav)
Suffix/prefix inputs with descriptions (title not read out on Voiceover)

**Comment by yatesn** [comment]
Gather updated requirements from Sarah Brooks (eQ) and contact craig.dunn@digitalaccessibilitycentre.org and/or cam.nicholl@digitalaccessibilitycentre.org
